### PR TITLE
Add missing "brief" entries to reuters

### PR DESCRIPTION
**Pull request by jbragg**
This brings the number of examples for ModApte to match the stated `Training set (9,603 docs)...Test Set (3,299 docs)`

### CODE REVIEW

**Comment by jbragg** [comment]
@lhoestq I ran `make style` but CI code quality still failing and I don't have access to logs

**Comment by jbragg** [comment]
It's also likely that due to the previous placement of the field initialization, much of the data about topics etc was simply wrong and carried over from previous entries. Model scores seem to improve significantly with this PR.

**Review by lhoestq — COMMENTED** [review]
Good catch ! And thank you for the fix :) 

Could you run `make style` to fix the CI check ?
EDIT: just saw you message above.

Let me run make style for you

**Review by lhoestq — APPROVED** [review]
The CI is ok now :)

thanks for the fix !
